app/src/main/res/values/strings.xml

```diff
@@ -85,4 +85,8 @@
     <string name="in_stock" translatable="false">In stock</string>
     <string name="out_of_stock" translatable="false">Out of stock</string>
 
+    <string name="email_error">Email must be in format example@gmail.com</string>
+    <string name="password_error">Password must be greater than 8 and have upper and lower character</string>
```

Comment:
upper lower special char number and greater than 8

Reply:
done